src/_playground/Routes.js

```diff
@@ -327,10 +343,20 @@ export class Routes extends Component {
             );
         });
 
+        const sideBarClasses = classnames({
+            'frDocs-Sidebar': true,
```

Comment:
If these are always true, you can just list them like this: https://github.com/SAP/fundamental-react/blob/master/src/Alert/Alert.js#L34